docs/README.md

```diff
@@ -9,8 +10,11 @@ This project is in the early stages of development.
 
 
 ## Setup
+Ensure you have the following dependencies installed:
+ - [`node`](https://nodejs.org/en/) `16.x`
+ - [`npm`](https://www.npmjs.com/)
 
-To start the application locally:
+### Local Deployment
 
 ```
 npm install
```

Comment:
Could we encourage npm ci here?

Reply:
I feel like for manual installs we might wanna still do `npm install`? Wdyt @idreyn @mathcolo ?

Reply:
I generally only use install for adding or updating a package since it can modify package-lock (while npm ci does not), but I don't feel strongly about it. Feel free to merge this in